### DISCUSSION TITLE
2 different Gallerys How to it best

### DISCUSSION THREAD

**Discussion by hoelzlmani** [immich-app/immich#14071] [General]
Hello
first i will say thank you to the whole immich team. The Software is really cool and a nearly perfect alternative zu Google.

I´m not sure how 2 do this best.
Have one big gallery with around 100.000 Pictures from all my Vaccations. Made automatic Album creation like
Holidays - Thailand 2004 - 01 Food and the Pics.... next is 02 Krabi and so on. then holidays - bali 2005 - 01 02 03...
This runs perfect for me.

Now i would like to have another big gallery with 150.000 Pictures.
Same Path (instead of Holiday now Nature) - Germany 2005 - 01 munich 02 hamburg.....Next Swiss 2006 - 01 bern 02 basel ....

The Problem is now. The automatic Gallery  made the Albums fine, but i have a mix Albumgallery with Holiday and Nature in one row!
It is not easy to find things. 

Should i make a new user for "Nature" pictures? How can i seperate it to a different location on my harddisk and is the automatic Album Generation working? The 2nd user ist not admin! 

All Pictures on an external USB Drive (mount) - it is not on the same drive like the Immich App!
The App runs on a seperate docker through Proxmox.

Plan B could be a completly seperate Immich Gallery, but for all the Updates it is not funny to do it always twice.

I hope you understand what i will do and you have a good workaround for me.

Thank you.

**Comment by Nika250** [comment]
The same question, maybe someone can help.
